src/test/java/edu/isi/oba/ObaUtilsTest.java

```diff
@@ -78,8 +78,10 @@ public void missing_file () throws OWLOntologyCreationException{
 
     @Test
     public void run() {
-        String ontology1 = "https://mintproject.github.io/Mint-ModelCatalog-Ontology/release/1.4.0/ontology.xml";
-        String ontology2 = "https://knowledgecaptureanddiscovery.github.io/SoftwareDescriptionOntology/release/1.5.0/ontology.xml";
+        //See: https://github.com/mintproject/Mint-ModelCatalog-Ontology/tree/master/release/1.8.0/ontology.owl
+        String ontology1 = "https://raw.githubusercontent.com/mintproject/Mint-ModelCatalog-Ontology/master/release/1.8.0/ontology.owl";
```

Comment:
Correct link should be https://mintproject.github.io/Mint-ModelCatalog-Ontology/release/1.8.0/ontology.owl